styles.css

```diff
@@ -240,22 +177,19 @@ body {
   text-decoration: none;
   font-weight: 500;
   font-size: 12px;
-  line-height: 16px;
   letter-spacing: 0.03em;
   background-color: #d9d9f5;
-  border-radius: 9px;
-  opacity: 0.5;
+  border-radius: 5px;
   color: #6070ff;
-  padding: 4px 12px;
+  flex-grow: 0;
+  padding: 0 4px 0 4px;
```

Comment:
- [ ] Kindly fix word wrappings in your cards to match the figma design. 
![image](https://user-images.githubusercontent.com/91486791/193356725-b1823197-b185-4233-8489-0b5f2e94cd57.png)